### PR TITLE
Escape carriage symbol

### PR DESCRIPTION
**Pull request by lgeiger**
/cc @rgbkrk

### CODE REVIEW

**Comment by rgbkrk** [comment]
Looking good. Care to export the `escapeCarriageReturn` function? I'd love to use it before it even reaches `ansi-to-react`.

**Comment by lgeiger** [comment]
I could also create a small npm package so we could use it in `transformime` and make a PR to `jupyter notebook`.

**Comment by rgbkrk** [comment]
> I could also create a small npm package so we could use it in transformime and make a PR to jupyter notebook.

That sounds great.

**Comment by lgeiger** [comment]
Published: https://github.com/lgeiger/escape-carriage

File: test/index-spec.js [hunk 1]
```diff
@@ -27,4 +27,10 @@ describe('Ansi', () => {
     expect(el.text()).to.equal('hello world');
     expect(el.html()).to.equal('<code><span>hello </span><span style="color:rgb(0, 187, 0);">wo</span><span style="background-color:rgb(187, 187, 0);color:rgb(0, 187, 0);">rl</span><span>d</span></code>');
   });
+
+  it('can handle carrigage symbol', () => {
```
Comment (rgbkrk) [inline test/index-spec.js]: Just noticed this typo. carrigage => carriage

**Comment by rgbkrk** [comment]
Feel free to ship this, sorry I didn't notice this again until I did some org-level cleanup.

**Comment by lgeiger** [comment]
👍 I forgot about this PR too

**Comment by rgbkrk** [comment]
I just remembered that I wanted to see this get worked into the stream processing in nteract/nteract so the strings are cleaned up before they even reach React.

**Comment by lgeiger** [comment]
Shipped 🚀

**Pull request merged**